### DISCUSSION TITLE
UnrealKaraulov, Why Am I Blocked? (New BSPGuy Inefficiency Problems)

### DISCUSSION THREAD

**Discussion by ToxicHolyGrenade** [community/community#142655] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

For no given reason, I cannot post any more issue notifications regarding New BSPGuy. I am blocked for no given reason form what I had seen. The issue in question, is that when it comes to searching elements like models in env_model for instance, there is no option available to search env_models by what model it uses for instance. What I propose for this sort of problem, is to introduce a metasearch function to find entities based on internal properties and what models, sounds, etc, are used for them. And some shortcut keys like Del for delete is not working, both for the ENT menu and the camera screen.

**Comment by UnrealKaraulov** [comment]
I don't remember the reason, but I haven't been updating New BSPGuy for a long time. You can create your own fork to fix these issues.
